crates/fluvio-controlplane-metadata/src/topic/config.rs

```diff
@@ -236,7 +236,7 @@ deduplication:
     age: 1m
   filter:
     transform:
-      uses: infinyon/dedup-filter@0.1.0
+      uses: fluvio/dedup-bloom-filter@0.1.0
```

Comment:
instead of hardcoding this,  Can we configure this?

Reply:
this is just for unit test

Reply:
if you mean about the `DEFAULT_DEDUP_FILTER`. that would be the part 2: https://github.com/infinyon/fluvio/issues/4387

for now we're on https://github.com/infinyon/fluvio/issues/4388

what do you think?